src/views/Results.vue

```diff
@@ -50,10 +50,14 @@
     </ul>
 
     <div id="results-printable">
-      <Score />
-      <Obligations />
-
       <div class="container-fluid">
+        <h2>{{ "Section 1: " + $t("riskLevel") }} {{ ": " + score[3] }}</h2>
```

Comment:
From the table of contents, the #score anchor is broken. it should point to this h2. Same for French.

Reply:
Fixed.